lib/EnsEMBL/REST/Model/Phenotype.pm

```diff
@@ -221,6 +221,7 @@ sub fetch_features_by_gene {
   unless (defined $gene_ad ) {Catalyst::Exception->throw("Species $species not found.");}
 
   my $phenfeat_ad = $c->model('Registry')->get_adaptor($species,'variation', 'phenotypefeature');
+  $phenfeat_ad->_include_ontology(1);
```

Comment:
On balance, perhaps _include_ontology() should not have an underscore prefix if it was intended to be called from outside the object. I won't ask you to change it here, but it is unconventional

Reply:
Thanks for checking @nerdstrike. Yes, well spotted and totally agree. Will sort this out in one of the future releases, as it involves deeper look into the API.